packages/flutter_tools/test/commands/build_flx_test.dart

```diff
@@ -0,0 +1,40 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/base/file_system.dart';
+import 'package:flutter_tools/src/base/process.dart';
+import 'package:test/test.dart';
+import '../src/context.dart';
+
+void main() {
+  group('build snapshot', () {
+    Directory temp;
+    Directory projectDir;
+
+    setUp(() {
+      temp = fs.systemTempDirectory.createTempSync('flutter_build_flx');
+      projectDir = temp.childDirectory('flutter_project');
+    });
+
+    tearDown(() {
+      temp.deleteSync(recursive: true);
+    });
+
+    testUsingContext('rebuilds if target changes', () async {
+      await runAsync(<String>['flutter', 'create', projectDir.path], allowReentrantFlutter: true);
```

Comment:
This'll only work if `flutter` is on the path (and maybe not on Windows). The other tests use `createTestCommandRunner()`, maybe that instead?